Podcast/EmptyStateView.swift

```diff
@@ -22,17 +23,21 @@ enum EmptyStateType {
             return "No Bookmarks"
         case .search:
             return "Sorry!"
+        case .feed:
+            return "No Feed"
         }
     }
     
     var explanation: String {
         switch self {
         case .pastSearch:
-            return "Find your favorite podcast episodes & series, friends, and tags."
+            return "Find your favorite podcast episodes, series, & friends."
         case .bookmarks:
             return "You can save podcast episodes for later here. Start looking now!"
         case .search:
             return "No results found."
+        case .feed:
+            return "Oh no! Your feed is empty. Follow series and people to get live updates!"
```

Comment:
also the empty state probably shouldn't show up while loading:
![image](https://user-images.githubusercontent.com/20246620/32148600-ebc550ce-bccf-11e7-9026-fa9835a0bcf5.png)

Reply:
Yes, I agree I have to fix this. I will fix this in another PR. I added the people addition because you can get feed updates from people you follow as well.

Reply:
I am going to update all the empty state views in another PR